src/IECoreImage/ClientDisplayDriver.cpp

```diff
@@ -42,7 +42,14 @@
 // This header needs to be here so that on Windows it doesn't fail with
 // winsock2.h included more than once, and under the above include so that it
 // doesn't fail on macOS as intrusive_ptr needs to be defined via RefCounted.h
-#include "boost/asio.hpp"
+#include <boost/version.hpp>
+#if BOOST_VERSION >= 106600
+#include <boost/asio.hpp>
+#include <boost/asio/io_context.hpp>
+#else
+#include <boost/asio.hpp>
+#include <boost/asio/io_service.hpp>
+#endif
```

Comment:
I don't think this change is necessary - in both boost versions, `asio.hpp` includes everything.